drivers/video/ov5640.c

```diff
@@ -1434,6 +1379,9 @@ static int ov5640_init(const struct device *dev)
 		.bus_type = OV5640_EP_PROP_OR(n, bus_type, VIDEO_BUS_TYPE_CSI2_DPHY),              \
 		.bus_width = OV5640_EP_PROP_OR(n, bus_width, 10),                                  \
 		.data_shift = OV5640_EP_PROP_OR(n, data_shift, 0),                                 \
+		.pclk_sample = OV5640_EP_PROP_OR(n, pclk_sample, 0),                               \
+		.hsync_active = OV5640_EP_PROP_OR(n, hsync_active, 0),                             \
+		.vsync_active = OV5640_EP_PROP_OR(n, vsync_active, 0),                             \
```

Comment:
Maybe some day we have a `struct video_endpoint_dt_spec` to automate all of it? :)